src/test/kotlin/com/terraformation/backend/db/DatabaseBackedTest.kt

```diff
@@ -0,0 +1,2482 @@
+package com.terraformation.backend.db
```

Comment:
Other than comments below, this is an exact copy of `DatabaseTest` before this PR.